test/controllers/bigbluebutton_api_controller_test.rb

```diff
@@ -1,6 +1,16 @@
 require 'test_helper'
+require 'redis_publisher'
 
 class BigbluebuttonApiControllerTest < ActionDispatch::IntegrationTest
+  def run
+    # Stub the Redis publisher calls, each test case expects maximun 2 calls.
+    mock = MiniTest::Mock.new
+    2.times.each { mock.expect :publish, true, [String, String] }
+    ::RedisPublisher.stub :redis, mock do
+      super
+    end
```

Comment:
You could have a test helper with this block so it's not repeated in other files.